### ISSUE TITLE
Editando notebooks

### ISSUE THREAD

**Issue by cpcenteno** [infiniemlabs-acustica/python_introduccion#20]
¿Cómo puedo editar un notebook de manera de poder copiar/pegar en otra aplicación? ¿Es posible?

**Comment by fchungo** [comment]
Buenas! Si te referís a copiar el código de un notebook a un archivo **.py**, es posible hacerlo.
Desde Jupyter Lab, se puede ir al menú **File** -> **Export Notebook as** -> **Executable Script**. Esto genera un archivo **.py** que podrías abrir y ejecutar desde Spyder, por ejemplo.

**Comment by cpcenteno** [comment]
No... me refería a exportarlo en formato .docx o .pdf, por ejemplo

**Comment by fchungo** [comment]
Retomo este tema, en la clase vimos que Jupyter requiere instalar **xelatex** para poder generar archivos pdf. En [este link](https://miktex.org/) se puede descargar el mismo. Otra posibilidad es exportarlo como **.md**, que es un formato de texto enriquecido. Para abrir los markdown hay dos opciones:
1) Usar la  extensión de Chrome: [Markdown preview](https://chrome.google.com/webstore/detail/markdown-preview/jmchmkecamhbiokiopfpnfgbidieafmd?hl=en)
2) Usar la página [mardown-to-pdf](url) para generar el pdf.
